lib/logging/GhostLogger.js

```diff
@@ -297,34 +297,30 @@ class GhostLogger {
      * logging.info({}, {}) --> is one object
      * logging.error(new Error()) --> is {err: new Error()}
      */
-    log(type, args) {
-        let modifiedArguments;
-
-        each(args, function (value) {
-            if (value instanceof Error) {
-                if (!modifiedArguments) {
-                    modifiedArguments = {};
-                }
-
-                modifiedArguments.err = value;
-            } else if (isObject(value)) {
-                if (!modifiedArguments) {
-                    modifiedArguments = {};
-                }
-
-                var keys = Object.keys(value);
-                each(keys, function (key) {
-                    modifiedArguments[key] = value[key];
-                });
-            } else {
-                if (!modifiedArguments) {
-                    modifiedArguments = '';
-                }
-
-                modifiedArguments += value;
-                modifiedArguments += ' ';
-            }
-        });
+     log(type, args) {
+         let modifiedMessages = [];
+         let modifiedObject = {};
+         let modifiedArguments = [];
+
+         each(args, function (value) {
+             if (value instanceof Error) {
+                 modifiedObject.err = value;
+             } else if (isObject(value)) {
+               var keys = Object.keys(value);
```

Comment:
Something is off with formatting here and no need to declare additional variable I think, `each(Object.keys(value), function (key) {` is rather readable